dvl-a50/static/index.html

```diff
@@ -130,10 +136,17 @@ <h2>Water Linked DVL Configuration Page</h2>
 				orientationOptions: {
 					"Downwards (LED pointing forward)": 1,
 					"Forward (Experimental)": 2,
-				}
+				},
+				confidenceCutoff: 10,
```

Comment:
Should this perhaps match the `DvlDriver` default? Or is it immediately overwritten by fetching from the backend or something? It would be weird if the displayed value doesn't match the internal one.